wgsl/index.bs

```diff
@@ -11866,3 +11866,18 @@ stage.
 `storageBarrier` affects memory and atomic operations in the [=address spaces/storage=] address space.
 
 `workgroupBarrier` affects memory and atomic operations in the [=address spaces/workgroup=] address space.
+
+## Assertion Built-in Functions ## {#assert-builtin-functions}
+
+WGSL provides the following assertion functions:
+
+```rust
+@const fn staticAssert(cond : bool)
+```
+
+`cond` [=shader-creation error|must=] be a [=creation-time expression=].
+
+The function produces a [=shader-creation error=] if `cond` evaluates to `false`.
```

Comment:
Is there a way to tell if I setup the `cond` to accidentally not be a creation time expression, or if it failed because it evaluated to `false`? Will it just be down to inspecting the error message returned?

Reply:
Generally yes, I think you'd have to inspect the error message. You could write the assertion slightly differently and produce a name for the condition if you wished to disambiguate those errors more easily. E.g.:
```
const test_cond = ....; // error if this is not a creation-time expression
staticAssert(test_cond);
```

So there would be an extra error in that example.